lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1471,30 +1475,17 @@ def _on_logging_relation_departed(self, _: RelationEvent):
         self.on.loki_push_api_endpoint_departed.emit()
 
     @property
-    def loki_push_api(self) -> List[dict]:
+    def loki_endpoints(self) -> List[dict]:
         """Fetch Loki Push API endpoints sent from LokiPushApiProvider through relation data.
 
         Returns:
             A list with Loki Push API endpoints.
         """
-        endpoints = []
+        endpoints = []  # type: list
```

Comment:
What does the type: list comment do? I'd imagine it would need to be identical to the return type.

Reply:
It makes pypy not annoyed with me